.travis.yml

```diff
@@ -1,5 +1,23 @@
 language: ruby
-script: bundle exec jekyll build
+
+sudo: required
+
+services:
+  - docker
+
+env:
+  DOCKER_COMPOSE_VERSION: 1.12.0
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
```

Comment:
Not really, last time i used travis.ci docker-compose was not yet a built-in command, fixed thanks.